core/src/banking_stage.rs

```diff
@@ -952,6 +956,8 @@ impl BankingStage {
                     "forward",
                 );
                 slot_metrics_tracker.increment_forward_us(forward_time.as_us());
+                // take metrics action after forwarding packets
+                slot_metrics_tracker.apply_action(metrics_action);
```

Comment:
The only difference here is that the reported slot times in here: https://github.com/solana-labs/solana/blob/c8dd942c810063942703c813e92a6730fa6383c8/core/src/leader_slot_banking_stage_metrics.rs#L273, specifically these three metrics https://github.com/solana-labs/solana/blob/c8dd942c810063942703c813e92a6730fa6383c8/core/src/leader_slot_banking_stage_timing_metrics.rs#L161-L171  will now have the additional latency of forwarding.

I think this is fine, since we also know how much time was spent in forwarding: https://github.com/solana-labs/solana/blob/c8dd942c810063942703c813e92a6730fa6383c8/core/src/leader_slot_banking_stage_timing_metrics.rs#L215-L216

Reply:
Good call! Implemented option 1 in 65014a5f58141964de6ec1f517b44f5a8a302a03